examples/pipeline-example.yaml

```
@@ -42,6 +42,33 @@
       - [batch2.input.training-labels*, train.input.training-labels]
       - [batch2.metadata.optimal_learning_rate, train.parameter.learning_rate]
       - [batch2.output.processed-images*, train.input.training-images]
+- pipeline:
+    name: My deployment pipeline
+    nodes:
+      - name: batch1
+        type: execution
+        step: Batch feature extraction
+      - name: train
+        type: execution
+        step: Train model
+      - name: deploy
+        type: deployment
+        deployment:
+          name: MyDeployment
+          version:
+            name: 2000102.0
+            branch: any
+            commit: none
+            endpoints:
+              - name: predict-digit
+                files:
+                  - name: model
+    edges:
+      #  node.entity_type.entity_name    node.entity_type.entity_name
+      - [batch1.input.training-labels*, train.input.training-labels]
+      - [batch1.metadata.optimal_learning_rate, train.parameter.learning_rate]
+      - [batch1.output.processed-images*, train.input.training-images]
+      - [train.output.model, deploy.file.model]
```

If there are multiple endpoints...
```suggestion
      - [train.output.model, deploy.file.predict-digit.model]
```